dataactcore/models/jobModels.py

```diff
@@ -117,10 +117,10 @@ class JobDependency(Base):
     __tablename__ = "job_dependency"
```

Comment:
Yeah we should. A job can't exist without a submission (because then it doesn't really serve a purpose) and if we're deleting everything related to a submission, that includes the related Jobs and their dependencies.